tests/test_queries.py

```diff
@@ -495,6 +495,59 @@ def test_select_non_projected(self):
         self.assertEqual(list(ret), [{"bar": 2}])
 
 
+class TestSaveJson(BaseSystemTest):
+
+    """ Tests that involve saving the results into a JSON file """
+
+    def setUp(self):
+        super(TestSaveJson, self).setUp()
+        self.test_file_name = 'out.json'
```

Comment:
We want to write to a temporary file ideally, however if the parser receives a full Windows path it blows up :-(

Reply:
I think that all you have to do to support a windows file path is to add `\` and `:` to the filename regex here:
https://github.com/stevearc/dql/blob/6b0ab8df07afd226816937ee98b4ef73187206fc/dql/grammar/common.py#L168

if you want to try that